pkg/gds/gds_test.go

```diff
@@ -51,14 +53,19 @@ func loadFixtures(s *gdsTestSuite) {
 		data, err := os.ReadFile(path)
 		require.NoError(err)
 		if strings.HasPrefix(info.Name(), "vasps::") {
-			// TODO: Unmarshal into VASP object when the fixtures are fixed.
 			vasp := &pb.VASP{}
 			err = protojson.Unmarshal(data, vasp)
 			require.NoError(err)
 			dbVASPs[info.Name()] = vasp
 			return nil
 		}
-
+		if strings.HasPrefix(info.Name(), "certs::") {
+			cert := &models.CertificateRequest{}
+			err = protojson.Unmarshal(data, cert)
+			require.NoError(err)
+			dbCerts[info.Name()] = cert
+			return nil
+		}
```

Comment:
This was pretty much how I was originally planning on adding the certs, although following discussion in #203 I think I'm going to end up making the dbVASPs/dbCerts into a generic map anyway, so you don't have to include these changes here if you don't want to (although it does help verify for the time being that the CertReq fixtures are valid).